File: pset1/cash/cash.c
#include <stdio.h>
#include <math.h>
#include <cs50.h>

int main(void)
{
    float dollars;
    // Prompts user for input in dollars
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);

    // Converts dollars to cent and round
    int cents = round(dollars * 100);

    // Counter to count coins used
    int coins = 0;

    // Starts counting num of coins using greedy algorithm
    while (cents != 0)
    {
        if (cents >= 25)
        {
            cents += - 25;
            coins++;
        }
        else if (cents >= 10)
        {
            cents += - 10;
            coins++;
        }
        else if (cents >= 5)
        {
            cents += -5;
            coins++;
        }
        else if (cents >= 1)
        {
            cents += -1;
            coins++;
        }
    }
    printf("%i\n", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    dollars = get_float("Change owed: ")
    if dollars >= 0:
        break

# Converts dollars to cent and round
cents = round(dollars * 100, 2)

# Counter to count coins used
coins = 0

# Starts counting num of coins using greedy algorithm
while cents != 0:
    if cents >= 25:
        cents += -25
        coins += 1

    elif cents >= 10:
        cents += -10
        coins += 1

    elif cents >= 5:
        cents += -5
        coins += 1

    elif cents >= 1:
        cents += -1
        coins += 1

print(coins)